include/constants/items.h

```diff
@@ -1002,6 +1005,6 @@
 #define ITEM_B_USE_OTHER    2
 
 // Check if the item is one that can be used on a Pokemon.
-#define ITEM_HAS_EFFECT(item) ((item) >= ITEM_POTION && (item) <= LAST_BERRY_INDEX)
+#define ITEM_HAS_EFFECT(item) ((item) >= ITEM_POTION && (item) <= ITEM_PEAT_BLOCK)
```

Comment:
why?

Reply:
Do you think it's time to finally get rid of `ITEM_HAS_EFFECT`? This macro limits what items can be used on Pokémon, and needed to be extended in order to the incoming items to work.

Reply:
It's a nuisance, so it sure would be good if it could be ditched.

Reply:
> It's a nuisance, so it sure would be good if it could be ditched.

Let's handle it in a separate PR.